gems/aws-sdk-s3/lib/aws-sdk-s3/presigner.rb

```diff
@@ -15,6 +15,7 @@ class Presigner
         'amz-sdk-request',
         'cache-control',
         'content-length', # due to a ELB bug
+        'content-type',
```

Comment:
Why are we doing this? And do presigned urls still work?

Reply:
The rest-xml protocol did not have content-type header in most cases... which caused failures in our protocol-tests. This PR adds this [here](https://github.com/aws/aws-sdk-ruby/blob/8df5f93263aa722e10e0fcad58bd729ef92ddf49/gems/aws-sdk-core/lib/aws-sdk-core/plugins/protocols/rest_xml.rb#L8).

We had integration tests for presigned urls that failed after the above addition. To remedy this, I updated here to blacklist the `content-type` header and the integration tests was successful. I plan to do a manual test of presigned url before PR release.

Reply:
You may need to change the integration tests but keep the behavior here. Content type is necessary for PUT/POST presigned requests. It should not be present for GET presigned urls.